src/main/java/io/stargate/sgv2/jsonapi/exception/playing/ErrorConfig.java

```diff
@@ -0,0 +1,327 @@
+package io.stargate.sgv2.jsonapi.exception.playing;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.LoadingCache;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * Java objects to hold the config from the errors config yaml file for use by {@link ErrorTemplate}
+ *
+ * <p>See the {@link #DEFAULT_ERROR_CONFIG_FILE} for description of what the yaml should look like.
+ *
+ * <p>To use this, call {@link #getInstance()} in code, this will cause the yaml file to be loaded
+ * if needed. The file will be loaded from {@link #DEFAULT_ERROR_CONFIG_FILE} resource in the JAR.
+ * Then use {@link #getErrorDetail(ErrorFamily, String, String)} if you want the template, or the
+ * use {@link #getSnippetVars()} when running templates.
+ *
+ * <p>If you need more control use the {@link #initializeFromYamlResource(String)} before any code
```

Comment:
... not necessarily easy/possible to do, due to class loading/initialization constraints.

Reply:
note to be - revisit the comment with how this is used in testing